src/js/components/DescriptionList.js

```diff
@@ -23,11 +23,14 @@ class DescriptionList extends React.Component {
 
       // Check whether we are trying to render an object that is not a
       // React component
-      if (typeof value === 'object' && !Array.isArray(value) &&
-        value !== null && !React.isValidElement(value)) {
+      if (typeof value === 'object' && value !== null &&
```

Comment:
It's transformed into an object (`Object.assign({}, value)`) so that we will get sublists for arrays as well. But @leemunroe is currently exploring different ways to render the configuration details and I might drop this change...